sky/clouds/cloud.py

```diff
@@ -190,26 +190,28 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
         raise NotImplementedError
 
     def get_current_user_identity(self) -> Optional[str]:
-        """(Advanced) Returns currently active user identity of on this cloud.
+        """(Advanced) Returns currently active user identity of this cloud.
 
         The user "identity" is associated with each SkyPilot cluster they
-        creates. This is used in protecting cluster operations in a multi-
-        identity scenario, where the same user/device can switch between
-        different cloud identities.
+        creates. This is used in protecting cluster operations, such as
+        provision, teardown and refresh status, in a multi-identity
+        scenario, where the same user/device can switch between different
+        cloud identities.
 
         The choice of what constitutes an identity is up to each cloud's
         implementation. In general, to suffice for the above purposes,
-        ensure that different identities imply different resources are
-        displayed in the cloud's console.
+        ensure that different identities should imply different sets of
+        resources are used when the user invoked each cloud's default
+        CLI/API.
 
         Example identities (see cloud implementations):
             - AWS: unique aws:user_id
             - GCP: email address + project ID
             - Azure: email address + subscription ID
 
         Returns:
-            None if the cloud does not have a concept of user identity;
-            otherwise the currently active user identity.
```

Comment:
keep the otherwise case?

Reply:
Oops, it was accidentally removed. Added back. Thanks!